File: user/nextMultiple.c
/*Считать с клавиатуры два целых числа - делимое и делитель, причем делитель не равен нулю.
Вывести на экран наименьшее число, кратное делителю, не меньшее делимого.*/

#include <stdio.h>
 
int main() {
    int dividend, divisor;
    int multiple;
    
    scanf("%d %d", &dividend, &divisor);
    
    multiple = dividend - dividend % divisor;
    if ( divisor < 0 ) {
        divisor *= -1;
    }
    
    if ( multiple > dividend ) {
        multiple -= divisor;
    }
    printf("%d\n", multiple);
    
    return 0;
}

File: bcw5/php/nextMultiple.php
<?php
	$dividend = fgets(STDIN);
	$divisor  = fgets(STDIN);
	$multiple;

	$multiple = $dividend - $dividend % $divisor;
    if ( $divisor < 0 ) {
        $divisor *= -1;
    }
    
    if ( $multiple < $dividend ) {
        $multiple += $divisor;
    }
    echo $multiple . PHP_EOL;
?>